pkg/printers/internalversion/describe_test.go

```diff
@@ -1398,3 +1399,61 @@ URL:	http://localhost
 		}
 	}
 }
+
+func TestDescribePodSecurityPolicy(t *testing.T) {
+	expected := []string{
+		"Name:.*mypsp",
+		"Allow Privileged:.*false",
+		"Default Add Capabilities:.*<none>",
+		"Required Drop Capabilities:.*<none>",
+		"Allowed Capabilities:.*<none>",
+		"Allowed Volume Types:.*<none>",
+		"Allow Host Network:.*false",
+		"Allow Host Ports:.*<none>",
+		"Allow Host PID:.*false",
+		"Allow Host IPC:.*false",
+		"Read Only Root Filesystem:.*false",
+		"SELinux Context Strategy: RunAsAny",
+		"User:.*<none>",
+		"Role:.*<none>",
+		"Type:.*<none>",
+		"Level:.*<none>",
+		"Run As User Strategy: RunAsAny",
+		"FSGroup Strategy: RunAsAny",
+		"Supplemental Groups Strategy: RunAsAny",
+		"Ranges:.*<none>",
+	}
+
+	fake := fake.NewSimpleClientset(&extensions.PodSecurityPolicy{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: "mypsp",
+		},
+		Spec: extensions.PodSecurityPolicySpec{
+			SELinux: extensions.SELinuxStrategyOptions{
+				Rule: extensions.SELinuxStrategyRunAsAny,
+			},
+			RunAsUser: extensions.RunAsUserStrategyOptions{
+				Rule: extensions.RunAsUserStrategyRunAsAny,
+			},
+			FSGroup: extensions.FSGroupStrategyOptions{
+				Rule: extensions.FSGroupStrategyRunAsAny,
+			},
+			SupplementalGroups: extensions.SupplementalGroupsStrategyOptions{
+				Rule: extensions.SupplementalGroupsStrategyRunAsAny,
+			},
+		},
+	})
+
+	c := &describeClient{T: t, Namespace: "", Interface: fake}
+	d := PodSecurityPolicyDescriber{c}
+	out, err := d.Describe("", "mypsp", printers.DescriberSettings{})
+	if err != nil {
+		t.Errorf("unexpected error: %v", err)
```

Comment:
Ideally this should be `t.Fatalf()` because if we fail here, we shouldn't inspect the output.